README.rst

```diff
@@ -478,6 +478,34 @@ There's usually no need and thus no simple way to change those settings,
 but if you have reason to do so you need to override ``get_project_settings``
 method of ``scrapyrt.core.CrawlManager``.
 
+
+LOGGING
+=======
+
+ScrapyRT supports Scrapy logging with some limitations.
+
+For each crawl it creates handler that's attached to the root logger and
+collects log records for which it can determine what spider object
+current log is related to. The only way to pass object to the log record is
+``extra`` argument (see explanation and another usage example `here
+<https://docs.python.org/2/library/logging.html#logging.debug>`_)::
+
+    logger.debug('Log message', extra={'spider': spider})
+
+Spider object is passed by default in `Spider.logger`_ and `Spider.log`_
+backwards compatibility wrapper so you don't have to pass it yourself
+if you're using them. All logs record that don't have reference to spider object
+or reference another spider object in the same process will be ignored.
+
+Spider logging setup in ScrapyRT happens only after spider object instantiation,
+so logging from ``Spider.__init__`` method as well as logging from
+middleware, pipeline or extension instantiation due to initialization order in Scrapy.
+
+Also ScrapyRT doesn't support `LOG_STDOUT`_ - you cannot write stdout to
+spider log in ScrapyRT because there's no way to filter such logs and they will
+appear in all log files for crawls that are running simultaneously.
```

Comment:
well, Scrapy supports this, this note was added to warn user that this wouldn't work in ScrapyRT. Whether or not to use print statements is up to user.